problemsets-mat223/linearalbegra.tex

```diff
@@ -3880,10 +3910,46 @@ \section*{Task 2.3: Pat and Jamie}
 	Consider $A=\mat{1&0&0\\0&1&0}$.
 	\begin{parts}
 		\item Describe the row space of $A$.
+			\begin{solution}
+				The row space of $A$ is $\Span\Set*{\mat{1\\0\\0},\mat{0\\1\\0}}$,
+				the $xy$-plane in $\R^3$. 
+			\end{solution}
 		\item Describe the column space of $A$.
+			\begin{solution}
+				The column space of $A$ is $\Span\Set*{\mat{1\\0},\mat{0\\1},\mat{0\\0}}
+					=\Span\Set*{\mat{1\\0},\mat{0\\1}}=\R^2$.
+			\end{solution}
 		\item Is the row space of $A$ the same as the column space of $A$?
+			\begin{solution}[inline]
+				No.
```

Comment:
Explain more. "No. Although they are both two dimensional spaces, all vectors in the row space of A have three coordinates (with the third one being zero) while vectors in the column space of A have two components. Thus these two spaces are different."